pkg/api/device_req_resp.go

```diff
@@ -0,0 +1,35 @@
+package api
+
+import "github.com/pborman/uuid"
+
+const (
+	// DeviceEnabled represents enabled
+	DeviceEnabled = "Enabled"
+
+	// DeviceFrozen represents frozen
+	DeviceFrozen = "Frozen"
+
+	// DeviceEvacuated represents evacuated
+	DeviceEvacuated = "Evacuated"
+
+	// DeviceFailed represents failed
+	DeviceFailed = "Failed"
+)
+
+// AddDeviceReq structure
+type AddDeviceReq struct {
+	PeerID uuid.UUID `json:"peer-id"`
+	Names  []string  `json:"names"`
```

Comment:
How about just name this as `Devices` and remove the below structs ? Devices shall always start at state `Disabled`.

Reply:
@prashanthpai I need to maintain state of each device for which I will need the other structs.

Reply:
Okay